### ISSUE TITLE
Stop and remove docker-containers after `crossplane render`

### ISSUE THREAD

**Issue by kaessert** [crossplane/crossplane#6028]
### What problem are you facing?

After running `crossplane render` command i found that it leaves behind running function-containers:
```
$> docker ps | grep crossplane-contrib/function | wc -l
     122
```

When doing development on composition, as you see this stacks up quite high, quite quickly. 

### How could Crossplane help solve your problem?

Stop and also remove containers.

**Comment by bobh66** [comment]
This is fixed by #5996 - the fix is on main but has not been released yet.

**Comment by kaessert** [comment]
@bobh66 I see it in the changelog https://github.com/crossplane/crossplane/releases/tag/v1.18.0, can we close this?
